packages/wdio-cli/src/interface.js

```diff
@@ -191,7 +192,7 @@ export default class WDIOCLInterface extends EventEmitter {
         /**
          * allow to exit repl mode via Ctrl+C
          */
-        if (this.inDebugMode) {
+        if (this.inDebugMode || this.endedNormally) {
```

Comment:
before this message was not printed in the very end because we used to kill process.

As far as we always call SIGTERM, we need to determine if process ended normally or Ctrl+C was really called.